index.d.ts

```diff
@@ -45,12 +48,13 @@ export class DDWAF {
 
   readonly diagnostics: {
     ruleset_version?: string,
-    rules?: diagnosticsInfo,
-    custom_rules?: diagnosticsInfo,
-    exclusions?: diagnosticsInfo,
-    rules_override?: diagnosticsInfo,
-    rules_data?: diagnosticsInfo,
-    processors?: diagnosticsInfo
+    rules?: diagnosticsError | diagnosticsInfo,
+    custom_rules?: diagnosticsError | diagnosticsInfo,
+    exclusions?: diagnosticsError | diagnosticsInfo,
+    rules_override?: diagnosticsError | diagnosticsInfo,
+    rules_data?: diagnosticsError | diagnosticsInfo,
+    processors?: diagnosticsError | diagnosticsInfo,
+    actions?: diagnosticsError | diagnosticsInfo
```

Comment:
Updated to match WAF output. 
See [ruleset_info](https://github.com/DataDog/libddwaf/blob/master/src/ruleset_info.hpp#L120)